tools/ports/freetype.py

```diff
@@ -67,10 +59,12 @@ def create(final):
             'src/pfr/pfr.c',
             'src/psaux/psaux.c',
             'src/pshinter/pshinter.c',
-            'src/psnames/psmodule.c',
+            'src/psnames/psnames.c',
             'src/raster/raster.c',
+            'src/sdf/sdf.c',
             'src/sfnt/sfnt.c',
             'src/smooth/smooth.c',
+            'src/svg/ftsvg.c',
```

Comment:
this should be `src/svg/svg.c`

- https://gitlab.freedesktop.org/freetype/freetype/-/blob/VER-2-12-1/CMakeLists.txt#L408
- https://gitlab.freedesktop.org/freetype/freetype/-/blob/VER-2-12-1/src/svg/svg.c

Reply:
Thanks for catching that. It's fixed.